pkg/distributors/contracts/MultiDistributor.sol

```diff
@@ -263,7 +263,7 @@ contract MultiDistributor is IMultiDistributor, ReentrancyGuard, MultiDistributo
         // will be merged. In both scenarios we round down to avoid paying more tokens than were received.
         if (block.timestamp >= periodFinish) {
             // Current distribution period has ended so new period consists only of amount provided.
-            distribution.paymentRate = Math.divDown(amount, duration);
+            distribution.paymentRate = FixedPoint.divDown(amount, duration);
```

Comment:
Let's add comments mentioning a) that `paymentRate` is a fixed point number, and b) that by doing fixed point division on two non-fixed point numbers, we get a fixed point result.